extension/manifest.json

```diff
@@ -22,6 +28,19 @@
     "48": "icon48.png",
     "128": "icon128.png"
   },
-  "web_accessible_resources": ["defaultStations.json", "icon48.png",
-                               "icon48-dark.png", "options.html"]
-}
+  "web_accessible_resources": [
+    "defaultStations.json",
+    "icon48.png",
+    "icon48-dark.png",
+    "options.html"
+  ],
+  "commands": {
+    "play/pause": {
+      "suggested_key": {
+        "default": "MediaPlayPause"
+      },
+      "description": "Toggle Play / Pause",
+      "global": true
```

Comment:
I just found http://www.omgchrome.com/chrome-google-music-media-keys/ for how users can modify their Chrome settings to make particular extensions only consume media key presses when Chrome is focused. How does the extension behave if you change global here to false?